06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md

```diff
@@ -0,0 +1,23 @@
+#### Output
```

Comment:
Great UI!

You might want to rethink having a `Create Index` button per file. What happens when we have 50 files uploaded?